js/main.js

```diff
@@ -16,9 +20,75 @@ $submitForm.addEventListener('submit', function (event) {
   $entry.title = $title.value;
   $entry.notes = $notes.value;
   $entry.photo = $photoInput.value;
-  $entry.nextEntryId = data.nextEntryId;
+  $entry.entryId = data.nextEntryId;
   data.nextEntryId++;
   data.entries.push($entry);
   $photo.setAttribute('src', 'images/placeholder-image-square.jpg');
   $submitForm.reset();
+  $viewForm.setAttribute('class', 'view-form hidden');
+  $viewEntries.setAttribute('class', 'view-entries');
+  location.reload();
```

Comment:
Oh nooooo 😱 

`location.reload()` should never be used, this reloads your page entirely, literally the most inefficient action you could take. If this is to make the new entry show up when you navigate to the entries view, the real answer is to generate a DOM tree for that entry in your `submit` event handler, then `prepend` that entry to the entries list before changing views. You will get the same effect without needing to reload the entire page..